apps/condo/domains/notification/schema/RegisterPushNotificationTokenService.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Generated by `createservice notification.RegisterPushNotificationTokenService --type mutations`
+ */
+
+const get = require('lodash/get')
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+
+const { PushToken: PushTokenAPI } = require('@condo/domains/notification/utils/serverSchema')
+const access = require('@condo/domains/notification/access/RegisterPushNotificationTokenService')
+
+const RegisterPushNotificationTokenService = new GQLCustomSchema('RegisterPushNotificationTokenService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterPushNotificationTokenInput { dv: Int!, sender: SenderFieldInput!, deviceId: String!, token: String!, serviceType: String! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterPushNotificationTokenOutput { id: String!, deviceId: String!, token: String!, serviceType: String!, user: ID }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canRegisterPushNotificationToken,
+            schema: 'registerPushToken(data: RegisterPushNotificationTokenInput!): RegisterPushNotificationTokenOutput',
+            resolver: async (parent, args, context) => {
+                const { data: { dv, sender, deviceId, token, serviceType } } = args
+                const userId = get(context, 'authedItem.id', null)
+                const attrs = {
+                    dv,
+                    sender,
+                    deviceId,
+                    token,
+                    serviceType,
+                    user: userId ? { disconnectAll: true, connect: { id: userId } } : null,
+                }
+                const [existingToken] = await PushTokenAPI.getFirst(context, { deviceId, serviceType })
```

Comment:
it's better to create getOrCreate helper.

Reply:
implemented